server/test.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+import os
+import cccp_reader
+import yaml
+
+
+def touch_file(file_path):
+    with open(file_path, 'a'):
+        os.utime(file_path, None)
+
+
+def _print_test_msg(msg):
+    print "=" * len(msg)
+    print msg
+    print "=" * len(msg)
+
+
+class Test01CCCPReader(unittest.TestCase):
+
+    def _gen_mock_cccp_yaml_data(self, jobid, test_skip, test_script, build_script, delivery_script):
+        return {
+            "job-id": jobid,
+            "test-skip": test_skip,
+            "test-script": test_script,
+            "build-script": build_script,
+            "delivery-script": delivery_script
+        }
+
+    def _setup_mock(self, cccp_yml_data=None):
+        self._target_file = "./test_file"
+        self._cccp_mock = "./mock_cccp.yml"
+        self._current_dir = os.getcwd()
+        self._build_script_path = None
+        self._test_script_path = None
+        self._delivery_script_path = None
+        touch_file(self._target_file)
+
+        if cccp_yml_data:
+            touch_file(self._cccp_mock)
+            with open(self._cccp_mock, "w") as f:
+                yaml.dump(cccp_yml_data, f)
+
+            if "build-script" in cccp_yml_data and cccp_yml_data["build-script"]:
```

Comment:
IMO, `if "build-script" in cccp_yml_data and cccp_yml_data["build-script"]` would yield same as `if cccp_yml_data.get("build-script")`